test/nn/conv/test_pna_conv.py

```diff
@@ -36,6 +36,16 @@ def test_pna_conv():
         assert torch.allclose(jit(x, adj.t()), out, atol=1e-6)
 
 
+def test_pna_divide_input():
+    x = torch.randn(4, 16)
+    edge_index = torch.tensor([[0, 0, 0, 1, 2, 3], [1, 2, 3, 0, 0, 0]])
+    deg = torch.tensor([0, 3, 0, 1])
+    conv = PNAConv(16, 32, aggregators, scalers, deg=deg, divide_input=True,
```

Comment:
Should we add `divide_input` to `__repr__` and test `str(conv)`?

Reply:
I am also not sure about that. It seems for other convs usually we don't add this kind of bool args to `__repr__`

Reply:
We do bool args sometime like `SignedConv`. Maybe @rusty1s could provide some suggestions on when we should add bool args to `__repr__`.

Reply:
I don't have a strong opinion. Usually, I prefer to only add important args to `__repr__` since otherwise the output of `print(torch.nn.Module(...))` becomes hard to parse.

Reply:
Cool. `divide_input` is not that important. Let's not add it.